app/controllers/admin/collections_controller.rb

```diff
@@ -305,11 +305,12 @@ def update_access(collection, params)
 
     collection.default_visibility = params[:visibility] unless params[:visibility].blank?
     collection.default_hidden = params[:hidden] == "1"
-    if Avalon::Configuration.controlled_digital_lending_enabled?
+    collection.default_enable_cdl = params[:cdl] == "1"
```

Comment:
I like using the ActiveModel casting for converting to a boolean.
```suggestion
    collection.default_enable_cdl = ActiveModel::Type::Boolean.new.cast(params[:cdl])
```

Reply:
I don't think that the ActiveModel casting is going to work in this case. If CDL is turned on at the system level, then it becomes impossible to turn it off for individual collections. Unchecking the box results in `params[:cdl]` being `nil` which gets cast to `nil`. The model sees that `collection.default_enable_cdl == nil` and falls back to the system setting.

There is potentially some html/javascript that could be setup in the view to force the unchecked box to send `0` as the param, which would be cast to `false`. But I don't know if that solution is any better/cleaner than leaving the `params[:cdl] == 1`.